reindex-data.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# Hat tip to Mohammadreza Alagheband
+
+reindex_init()
```

Comment:
Cool script, you could also implement a curl script that would use `_reindex` api query, which is a bit faster than reindexing through logstash. Especially if you increase shard count
```json
"settings": {
    "index.refresh_interval": "5s",
    "index": {
      "number_of_shards": 15,
      "number_of_replicas": 0
    }
  },
```
https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-reindex.html